### ISSUE TITLE
Clarifying thermal heat flux at base of ice sheet

### ISSUE THREAD

**Issue by martinjuckes** [cmip6dr/CMIP6_DataRequest_VariableDefinitions#405]
The variable `hfegoubed` title "Geothermal Heat Flux Beneath Land Ice" and description "Upward geothermal heat flux per unit area beneath land ice".
   
The standard name, "upward_geothermal_heat_flux_at_ground_level_in_land_ice", is described as being "upward heat flux at the interface between the ice and bedrock".
    
My question concerns the precise description of the "interface between ice and bedrock", taking into account the facts that the ISMIP6 models appear to represent a horizontal transport of heat at this interface due to transport by melt water and that there should be (I think) a discontinuity in the thermal heat flux at the interface when there is melting.
   
 With "hfgeoubed", does ISMIP6 want the heat flux coming out of the rock or the heat flux going into the ice?
   
This question is referenced in a CF Discussions issue about the definition of "bedrock" in CF ([#2](https://github.com/cf-convention/vocabularies/issues/8)).

**Comment by martinjuckes** [comment]
Response from Sophie Nowicki:

Thanks for following this up... We are interested in the heat flux going into the ice. You are correct that some models may have complex models between the ice and the bedrock, but the majority do not... In order to minimize our data request, we wanted only the heat flux going into the ice.

If I look at the ISMIP6 data request on our wiki, we have the following name for hfgeoubed "upward_geothermal_heat_flux_in_land_ice", but you are right that the CF name on the CF convention page is "upward_geothermal_heat_flux_at_ground_level_in_land_ice"... I tried to dig up past discussions on this topic, but not having much luck...

In short, we are interested in the quantity going into the ice, and can happily change our data request name...

**Comment by martinjuckes** [comment]
Description modified to "Upward geothermal heat flux per unit area into the base of grounded land ice. This is related to the geothermal heat flux out of the bedrock, but may be modified by horizontal transport due to run-off and by melting at the interface." in 01.00.32
